tests/r7rs/r7rs-tests.scm

```diff
@@ -37,6 +38,13 @@
 ;; however (chibi test) provides nicer output, timings, and
 ;; approximate equivalence for floating point numbers.
 
+(define-syntax test
+  (syntax-rules ()
+    ((_ expected expr)
+      (test-equal (quote expected) expected expr))
```

Comment:
Did you mean `(quote expected)` => `expected` as we see below? It seems most of tests have `quote` already where required.